### PR TITLE
Separate js files

### PR DESCRIPTION
**Pull request by ramyabuva**

### CODE REVIEW

File: src/main/webapp/calendarScript.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      scheduleView: ['time'],
+      taskView: false,
+      week: {
+        workweek: true,
+      },
+    });
+  }
+
+  initCalendar();
+});
```
Comment (savsa) [inline src/main/webapp/calendarScript.js]: nit: need a new line at end of file

File: src/main/webapp/selectScript.js [hunk 2]
```diff
@@ -97,24 +92,17 @@ window.addEventListener('load', () => {
...
-  }
 
-  initCalendar();
-});
+  const CollegePlanner = {
+    selected: selected,
+    courses: courses,
+  };
+  return CollegePlanner;
+})();
```
Comment (savsa) [inline src/main/webapp/selectScript.js]: newline here too

File: src/main/webapp/vendor/startbootstrap/index.html [hunk 3]
```diff
@@ -252,7 +252,10 @@ <h5 class="modal-title" id="exampleModalLabel">Ready to Leave?</h5>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.20.1/moment.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/chance/1.0.13/chance.min.js"></script>
   <script src="https://uicdn.toast.com/tui-calendar/latest/tui-calendar.js"></script>
-  <script src="script.js"></script>
+
+
```
Comment (savsa) [inline src/main/webapp/vendor/startbootstrap/index.html]: nit: take out new line here

**Review by savsa — APPROVED** [review]

File: src/main/webapp/selectScript.js [hunk 4]
```diff
@@ -11,25 +11,20 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-window.addEventListener('load', () => {
+export const CollegePlanner = (function() {
```
Comment (wli) [inline src/main/webapp/selectScript.js]: arrow function: `() => {`

File: src/main/webapp/selectScript.js [hunk 5]
```diff
@@ -11,25 +11,20 @@
...
-    addToSelected();
-  });
-
   /**
    * Gets courses from /courselist servlet to populate dropdown list
    */
   async function getOptions() {
     const response = await fetch('/courses');
-    courseList = await response.json();
+    const courseList = await response.json();
```
Comment (wli) [inline src/main/webapp/selectScript.js:23]: hm, i feel like linters should have caught this. wonder if we need to fix something up later.

File: src/main/webapp/selectScript.js [hunk 6]
```diff
@@ -97,24 +92,17 @@ window.addEventListener('load', () => {
...
-    });
-  }
 
-  initCalendar();
-});
+  const CollegePlanner = {
+    selected: selected,
+    courses: courses,
+  };
+  return CollegePlanner;
```
Comment (wli) [inline src/main/webapp/selectScript.js]: what happens if you just do: 

```
return {
  selected: selected,
  courses: courses,
};
```

File: src/main/webapp/vendor/startbootstrap/index.html [hunk 7]
```diff
@@ -252,7 +252,10 @@ <h5 class="modal-title" id="exampleModalLabel">Ready to Leave?</h5>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.20.1/moment.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/chance/1.0.13/chance.min.js"></script>
   <script src="https://uicdn.toast.com/tui-calendar/latest/tui-calendar.js"></script>
-  <script src="script.js"></script>
+
+  <!-- Our scripts -->
+  <script type="module" src="selectScript.js"></script>
```
Comment (wli) [inline src/main/webapp/vendor/startbootstrap/index.html:257]: this is cool! Just note that this is a relatively recent feature: https://caniuse.com/#feat=mdn-javascript_statements_import

You probably wouldn't want to depend on this for a real production website just yet, but it works perfectly fine for our use cases here.

**Review by wli — APPROVED** [review]
Thanks for figuring this out!

**Review by triplehair — APPROVED** [review]

**Pull request merged**